src/harness/sas.py

```diff
@@ -228,7 +228,7 @@ def InjectEscZone(self, request):
     return _RequestPost('https://%s/admin/injectdata/esc_zone' % self._base_url,
                         request, self._tls_config)
 
-  def InjectExZone(self, request):
+  def InjectExclusionZone(self, request):
     return _RequestPost('https://%s/admin/injectdata/ex_zone' % self._base_url,
```

Comment:
Please use the correct URL.

Reply:
Modified   url to admin/injectdata/exclusion_zone